assembly/pom.xml

```diff
@@ -108,6 +108,13 @@
                     <value>parcel</value>
                 </property>
             </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>com.splicemachine</groupId>
+                    <artifactId>splice_sentry-${envClassifier}</artifactId>
+                    <version>2.5.0.1833-SNAPSHOT</version>
```

Comment:
Use ${project.version} here and fix other occurrences of 2.5.0.1833-SNAPSHOT in dependencies as well.

Reply:
The current version of the module has been changed to 2.5.0.1834-SNAPSHOT. This PR would introduce a new dependency on 2.5.0.1833-SNAPSHOT, which would be wrong and which would not be updated by the update script when the version changes to 1835. This is the reason for my request.